tensorbay/client/version.py

```diff
@@ -582,3 +582,101 @@ def delete_tag(self, name: str) -> None:
         delete_data: Dict[str, Any] = {"name": name}
 
         self._client.open_api_do("DELETE", "tags", self._dataset_id, json=delete_data)
+
+
+class JobMixin:  # pylint: disable=too-many-public-methods
+    """A mixin class supporting asynchronous jobs.
+
+    Arguments:
+        _dataset_id: Dataset ID.
+        _client: The client to interact between local and TensorBay.
+        _status: The version control status of the dataset.
+
+    """
+
+    _dataset_id: str
+    _client: Client
+    _status: Status
+
+    def _create_job(  # pylint: disable=too-many-arguments
+        self,
+        title: str,
+        job_type: str,
+        arguments: Dict[str, Any],
+        description: str = "",
+        max_retries: int = 3,
```

Comment:
Max reties means that the job will retry automatically with the given number?

Reply:
Yes. Considering that this function of the backend is not perfect, we will not launch this function now.